addon/components/snippet-plugin/snippet-insert.gts

```diff
@@ -75,9 +75,11 @@ export default class SnippetInsertComponent extends Component<Sig> {
         (tr, snippet) => tr.replaceSelectionWith(snippet),
```

Comment:
Do we need to do a `tr.setMeta('insertSnippet', true);` somewhere in this branch?

Reply:
It's on the createAndInsertSnippet function